index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="port-folio/main.css">
+    <title>Arksel's Portfolio</title>
+</head>
+<body>
+    <div class="mobile-container">
+            <nav class="navbar">
+                <div class="header">Logo</div>
+
+                <div class="menu">
+                    <button type="button" class="menu-btn"></button>
+                    <button type="button" class="menu-btn"></button>
+                    <button type="button" class="menu-btn"></button>
+                </div>
+
+            </nav>
+        <div class="main">
+            <div>
+                <h1>
+                    I'm Godfrey <br>
+                    Glad to see you!
+                </h1>
+                <p>
+                    I'm a software developer! I can help you build a product , 
+                    feature or website Look through some of my work and experience!
+                    If you like what you see and have a project you need coded, 
+                    don't hestiate to contact me.  
+                </p>
+            </div>
+        </div>
+        <div class="links">
+            <h2>LET'S CONNECT</h2>
```

Comment:
- [ ] The 'LET'S CONNECT' should be wrapped in an anchor(`a`) tag instead of an 'h2' tag. This is because it is a link 👍